blog/2024-09-11-swiss-statistics-days/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+slug: sst-2024
+title: |
+  Swiss Statistics Meeting - Renku: a platform for transparent and reproducible data-centric work
+authors: [elisabet, rokroskar, laura, jimena]
+tags:
+  - Swiss Statistics Meeting
+  - Poster
+image: ./image_1.png
+description: |
+  Renku was presented in the Swiss Statistics Meeting, which took place on Sept. 4th
+  and 5th 2024 in Aarau (Switzerland) and brought together specialists related to 
+  statistics domain across private and public sectors. The SDSC contribution “Renku: a platform 
+  for transparent and reproducible data-centric work” focused on those features in 
+  Renku 2.0 that make it a unique platform to support transparent data-centric work, 
+  since it connects data sources and code repositories, and enables launching sessions, 
+  where data scientists, statisticians and other practitioners can work in their prefererred
+  compute environment (Python, R, Matlab, etc.) and share their work.
+---
+
+![Renku in Swiss Statistics Meeting 2024](./image_1.png)
+
+<!-- truncate -->
```

Comment:
Honestly, you should truncate after the first paragraph. This does not look engaging at all:

![Screenshot 2024-09-23 at 09 18 44](https://github.com/user-attachments/assets/aaac6811-07a4-42c9-b5d3-f65cfcbe0401)